src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,20 @@
+package lotto.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    public static Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
   private static Scanner scanner = new Scanner(System.in);
```

클래스 내부에서만 사용하는 변수나 메소드는 꼭 접근제한자 수준을 줄여 주세요 😄